ios/Classes/SwiftPolarPlugin.swift

```diff
@@ -233,7 +467,7 @@ public class SwiftPolarPlugin:
                 }
                 exercises.append(data)
             }, onError: { error in
-                result(FlutterError(code: "Error listing exercises", message: error.localizedDescription, details: nil))
+                result(FlutterError(code: "Error listing exercises", message: error.localizedDescription, details: error.localizedDescription.description))
```

Comment:
Isn't this a debug description of `error.localizedDescription` object? Is this useful?